bundle/manifests/submariner.clusterserviceversion.yaml

```diff
@@ -103,61 +122,68 @@ spec:
     For complete information about subctl, please refer to [this page](https://submariner.io/operations/deployment/subctl).
     In addition to Operator and subctl, Submariner also provides [Helm Charts](https://submariner.io/operations/deployment/helm).
 
-    #### Deploying with subctl
-    Install the tool:
-    ```bash
-    curl -Ls https://get.submariner.io | bash
-    export PATH=$PATH:~/.local/bin
-    echo export PATH=\$PATH:~/.local/bin >> ~/.profile
-    ```
-    Deploy the broker:
+    On the broker cluster:
+    1. Create the broker namespace called `submariner-k8s-broker`.
+    2. Apply the broker RBAC definitions:
 
-    `subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER>`
+        `kubectl apply -k https://github.com/submariner-io/submariner-operator/tree/release-0.9/config/broker`
 
-    For each cluster you want to join, issue the following command:
+    3. Install the submariner operator from the catalog.
```

Comment:
```suggestion
    3. Install the Submariner Operator from the catalog.
```

Reply:
Should we provide a link?

Reply:
The description appears on the catalog web page. The user is already on the right place and just needs to hit the install button.